pkg/acl/acl.go

```diff
@@ -244,16 +151,15 @@ func normalizeOperation(operation string) (string, error) {
 
 func ApplyAccessControlPolicies(ctx context.Context, operation string, httpVerb commonv1pb.HTTPExtension_Verb, isHTTP bool, acl *config.AccessControlList) (bool, string) {
 	// Apply access control list filter
-	spiffeID, err := GetAndParseSpiffeID(ctx)
+	spiffeID, ok, err := spiffe.FromGRPCContext(ctx)
 	if err != nil {
-		// Apply the default action
-		log.Debugf("Error while reading spiffe id from client cert: %v. applying default global policy action", err)
+		log.Debugf(err.Error())
```

Comment:
Leftover log? Or should this at least have some message around it?

Reply:
Yeah, the message describing the action has been moved to the log line below. I'll add context to the error here :+1:

Reply:
Ok. Also change `Debugf` to `Debug` if there's no formatting. Otherwise, `%` chars inside the error message are interpreted as formatting things